### DISCUSSION TITLE
Setup base46 themes

### DISCUSSION THREAD

**Discussion by AjayShanker-geek** [LazyVim/LazyVim#5267] [Q&A]
<img width="1910" alt="Screenshot 2024-12-28 at 4 13 39 PM" src="https://github.com/user-attachments/assets/ae5c60a5-142b-4d55-beb5-a21a70b393be" />

Hi, is there a working method to use base46 with lazyvim? This is from following base46 readme setup and it seems it is not working correctly. Cheers!

**Comment by aniaan** [comment]
You can only integrate manually, and you can refer to the code of [cosmos-nvim](https://github.com/yetone/cosmos-nvim/blob/main/lua/layers/ui/colors.lua).

**Reply by AjayShanker-geek** [reply]
Might look into overwriting tokyonight theme.
